src/app/services/breathecode.js

```diff
@@ -8,92 +8,92 @@ class BreatheCodeClient {
     return {
```

Comment:
Check the `admissions()` function with `ESlint` enabled, it says something like 
"`Parsing error: unexpected token =`"
this problem is because this file works with `class` component and the function expects to be
```js
admissions() {
    return {...}
}
```
this problem also occurs with
```js
// error
marketing = () => ({...})

//expect solution
marketing() {
    return {...}
}

// expect the same solution in the functions
feedback = () => ({...})
certificates = () => ({...})
events = () => ({...})
```